utils/load_image.py

```diff
@@ -0,0 +1,90 @@
+#%%
+import tarfile
+from io import BytesIO
+from PIL import Image
+import os
+from pathlib import Path
+import pandas as pd
+
+
+basefolder_loc = Path(__file__).parents[1]
+TARFILE = tarfile.open(
+    os.path.join(basefolder_loc, "1.download-data", "data", "training.tar.gz"), "r:gz",
+)
+
+
+def load_img(
```

Comment:
I have tested this function with:

```python
from utils.load_image import load_img
load_img()
```

output:

![image](https://user-images.githubusercontent.com/7340421/70853710-96ecc280-1e7f-11ea-899d-6675ed4a08d8.png)

Reply:
I also tested:

```python
load_img(
    target="dopaminereceptor",
    plate="P1",
    cell_id=5,
    replicate=2,
    wellName="B11",
    field=1
)
```

output: 

![image](https://user-images.githubusercontent.com/7340421/70853749-40cc4f00-1e80-11ea-9eb2-261b7c236123.png)

Reply:
That looks good! (although they are so dark I needed a moment to see there are not completely black)